spec/lib/vanagon/platform/windows_spec.rb

```diff
@@ -38,6 +57,105 @@
           expect(cur_plat._platform.target_user).to eq(plat[:target_user])
         end
       end
+
+      describe '#generate_msi_packaging_artifacts' do
+        before(:each) do
+          # Create Workdir and temp root directory
+          FileUtils.mkdir_p("#{workdir}/wix")
+          FileUtils.mkdir_p("#{VANAGON_ROOT}/resources/windows/wix")
+          FileUtils.mkdir_p("#{PROJ_ROOT}/resources/windows/wix")
+          # Need to set proj_root to our fixtures/sample directory
+          cur_plat._platform.proj_root = PROJ_ROOT
+        end
+        after(:each) do
+          # Cleanup the complete work directory tree
+          FileUtils.rm_rf("#{WORK_BASE}")
+        end
+
+        it "Copies Wix File from product specific directory to output directory" do
+          # setup source directories and run artifact generation
+          FileUtils.cp("#{wixtestfiles}/file-1.wxs", "#{PROJ_ROOT}/resources/windows/wix/file-1.wxs")
```

Comment:
Is there a reason we can't just set whatever directory our fixtures are currently living in as the `project_root` and `vanagon_root`s, since we're setting those already?